### ISSUE TITLE
Failure to update 'version' number

### ISSUE THREAD

**Issue by BeerStud** [texstudio-org/texstudio#1433]
I have downloaded and installed version 3.0.4 twice. In both cases, the 'ABOUT" shows version 3.0.1. The program warns of a newer version even though I am supposed to have the newer version.

### Environment
- TeXstudio: TeXstudio 3.0.1 (git 3.0.1)
- Qt: 5.12.1, compiled with Qt 5.12.1 R
- OS: Windows 10 Pro, VERSION: 20H2, OS BUILD: 19042.746
- TeX distribution: MiKTeX

### Expected behavior
The Program would display the proper version

### Actual behavior
The program displays an incorrect version.

### How to reproduce
Download and install the newer 3.0.1 version from <https://github.com/texstudio-org/texstudio/releases/download/3.0.4/texstudio-3.0.4-win-qt5.exe>

**Comment by sunderme** [comment]
I just rechecked. Version number in about dialog is 3.0.4.

**Comment by BeerStud** [comment]
This is a screen capture of the "About" screen.
![TeXstudio-ScreenCapture](https://user-images.githubusercontent.com/17906160/104860368-38530d00-58f9-11eb-8116-d3ee0b690dd9.png)

**Comment by muzimuzhi** [comment]
@BeerStud Just a guess. Perhaps 3.0.4 is never successfully installed on your computer ... Have you tried to uninstall that 3.0.1?

**Comment by BeerStud** [comment]
I had to resort to using Revo Uninstaller <https://www.revouninstaller.com/> to completely remove TeXstudio before I could successfully install the new version. For whatever reason, the old version was not being completely over-written, or else something was getting screwed up in the Window's registry. In any case, it is fixed now.

It is a good thing I had a backup of my custom 'templates'. They were removed along with everything else. ✔
